src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Conversion.cs

```diff
@@ -325,6 +325,15 @@ private static bool IsFloatPointExpressionOfUnknownPrecision(BoundExpression rew
                     Debug.Assert(constantValueOpt == null);
                     return _dynamicFactory.MakeDynamicConversion(rewrittenOperand, explicitCastInCode || conversionKind == ConversionKind.ExplicitDynamic, isArrayIndex, @checked, rewrittenType).ToExpression();
 
+                case ConversionKind.ImplicitTuple:
+                    return RewriteTupleConversion(
+                        syntax: syntax,
+                        rewrittenOperand: rewrittenOperand,
+                        conversionKind: conversionKind,
+                        @checked: @checked,
+                        explicitCastInCode: explicitCastInCode,
+                        rewrittenType: (TupleTypeSymbol)rewrittenType);
```

Comment:
ok, having an issue to decide on this later, I will add code to handle underlying types by default, although in many places it does not look like it could be reached.